pyogrio/raw.py

```diff
@@ -425,7 +422,8 @@ def open_arrow(
             "geometry_type": "<geometry_type>",
             "geometry_name": "<name of geometry column in arrow table>",
         }
-    """
+
+    """  # noqa: E501
```

Comment:
Or just add line breaks to docstring where needed?

Reply:
The lines that are too long are in the code samples, not sure how to deal with those?

Reply:
Could just introduce a variable which would make those shorter:

```Python
    >>> with open_arrow(path, use_pyarrow=True) as source:
    >>>     meta, reader = source
    >>>     geom_col = meta["geometry_name"] or "wkb_geometry"
    >>>     for batch in reader:
    >>>         geometries = shapely.from_wkb(batch[geom_col])
```